### PR TITLE
Update common python package work-around steps for installing on Grav…

### PR DESCRIPTION
**Pull request by geoffreyblake**
*Description of changes:*
Update the work-around descriptions for installing python package Pillow, and add the work-around steps for the cffi package 
 on graviton.

### CODE REVIEW

File: python.md [hunk 1]
```diff
@@ -125,22 +125,41 @@ to the number of threads to use, the default is to use a single thread.
 
 ## 3. Other common Python packages
 
+### 3.1 cffi
```
Comment (AWSNB) [inline python.md:128]: @geoffreyblake should we make cffi as part of the prerequisite for all this

File: python.md [hunk 2]
```diff
@@ -125,22 +125,41 @@ to the number of threads to use, the default is to use a single thread.
...
-sudo apt-get install libtiff5-dev libjpeg8-dev libopenjp2-7-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python3-tk libharfbuzz-dev libfribidi-dev libxcb1-dev
-pip3 install --user pillow
+sudo apt-get install libffi-dev build-essential
+pip3 install --user cffi
 ```
 
+### 3.2 Pillow
+
+As of June 30, 2020, Pillow 7.2.0 was released as a binary wheel for Arm platforms, but contains a bug that prevents it from loading
+properly on distributions that use 64kB pages and must be compiled from source, i.e. Centos8 and Redhat8.  To determine if you need to
```
Comment (AWSNB) [inline python.md:147]: on *Linux* distributions

File: python.md [hunk 3]
```diff
@@ -125,22 +125,41 @@ to the number of threads to use, the default is to use a single thread.
...
+install Pillow from source, do the following:
+
+On **RedHat**:
```
Comment (AWSNB) [inline python.md:150]: RedHat and Centos

File: python.md [hunk 4]
```diff
@@ -125,22 +125,41 @@ to the number of threads to use, the default is to use a single thread.
...
+
+```bash
+if [[ $(getconf PAGESIZE) > 4096 ]]; then
+    sudo yum install libtiff-devel libjpeg-devel openjpeg2-devel zlib-devel freetype-devel lcms2-devel libwebp-devel tcl-devel tk-devel harfbuzz-devel fribidi-devel libraqm-devel libimagequant-devel libxcb-devel
+    pip3 install --user --no-binary pillow
+else
+    pip3 install --user pillow
+fi
+```
+
```
Comment (AWSNB) [inline python.md:160]: its not clear that ubuntu/amazonlinux doesnt need any of this.     so i would put "On **Ubuntu and AmazonLinux2**: 
 just follow normally install pip3 --user pillow"

File: python.md (hunk 1 above)
Comment (geoffreyblake) [reply python.md]: Updated this to include libffi-dev/libffi-devel in the general requirements at the top of the document.

File: python.md (hunk 3 above)
Comment (geoffreyblake) [reply python.md]: Done.

File: python.md (hunk 2 above)
Comment (geoffreyblake) [reply python.md]: Done

File: python.md (hunk 4 above)
Comment (geoffreyblake) [reply python.md:146]: Done.

**Pull request merged**